### ISSUE TITLE
Display the gateway language progress information as tabs on the language detail page

### ISSUE THREAD

**Issue by Redmayspecial** [unfoldingWord-dev/translationDatabaseWeb#447]
Add the progress as tabs under the language detail page.

![image](https://cloud.githubusercontent.com/assets/11600863/13325874/b026d1c4-dbb1-11e5-949f-c33de4635cc2.png)

![image](https://cloud.githubusercontent.com/assets/11600863/13325919/e5c58078-dbb1-11e5-9f16-cb755ff1a22b.png)

**Comment by rbnswartz** [comment]
Blocked until I can get a more recent database for testing.

**Comment by rbnswartz** [comment]
Waiting on merge.

**Comment by rbnswartz** [comment]
Merged into production
